packages/web-react/src/components/TextFieldBase/TextFieldBase.tsx

```diff
@@ -23,6 +23,12 @@ export const TextFieldBase = (props: SpiritTextFieldBaseProps) => {
       propertyName: 'validationState',
     },
   });
+  useDeprecationMessage({
+    method: 'custom',
+    trigger: !!(props?.message && !props?.validationState),
+    componentName: props?.isMultiline ? 'TextArea' : 'TextField',
+    customText: 'Message without validation status will be removed in the next version. Use `helperText` instead.',
```

Comment:
```suggestion
    customText: 'message prop without validationState prop will be unsupported in the next version. Use `helperText` instead.',
```

What do you think? It sounds better to me like this.

Reply:
fixed